client/homescreen/layout.js

```diff
@@ -25,6 +25,7 @@ import '../dashboard/style.scss';
 import TaskListPlaceholder from '../task-list/placeholder';
 import InboxPanel from '../inbox-panel';
 import { WelcomeModal } from './welcome-modal';
+import ActivityPanel from './activity-panel';
```

Comment:
Could we move towards always using named exports from now on? Its error prone to use defaults because its very easy to rename something different to what it was exported as and that makes it much harder to find usages in your IDE.